test/Conversion/TensorToXsmm/tensor-to-unary.mlir

```diff
@@ -32,12 +32,8 @@ func.func @pack3(%in: tensor<8x2x2x2xf32>, %out: tensor<2x2x1x4x2x2xf32>)-> tens
   return %2: tensor<2x2x1x4x2x2xf32>
 }
 
-// CHECK: func.func @pack3(%[[ARG0:.+]] memref<8x2x2x2xf32>, %[[ARG1:.+]] memref<2x2x1x4x2x2xf32>) -> memref<2x2x1x4x2x2xf32> {
-// CHECK: scf.for
-// CHECK:   scf.for
-// CHECK:     scf.for
-// CHECK:       memref.subview
-// CHECK:       linalg.transpose
-// CHECK:       memref.subview
-// CHECK:       %[[DISPATCH:.+]] = xsmm.unary.dispatch identity [2, 2, 2, 2] flags = (none) data_type = f32 
-// CHECK:       xsmm.unary identity(data_type = f32, %{{[^:]+}}, {{[^:]+}}, {{[^:]+}}) : (i64, memref<2x2xf32>, memref<2x2xf32, strided<[2, 1], offset: ?>>) -> ()
+// CHECK-LABEL: @pack3
```

Comment:
What happened here? Why is it not detecting xsmm calls any more?

This seems a repeated pattern throughout the tests. We must make sure that this is really what we want.

@KavithaTipturMadhu